Bindings/Java/Matlab/Utilities/osimTableToStruct.m

```diff
@@ -78,12 +79,24 @@
     % Get the osim table column label
     col_label  = char(osimtable.getColumnLabels.get(iLabel));
     disp_label = col_label;
-    % MATLAB structs must start with a letter, and can only contain
-    % letters, digits, and underscores.
-    illegalChars = [{' '} {'*'} {'.'} {'/'} {'-'} {'|'}];
-    for i = 1 : length(illegalChars)
-        if ~isempty(strfind(col_label, illegalChars{i}))
-           col_label = strrep(col_label,illegalChars{i}, '_'); 
+    % MATLAB variable names must start with a letter, can only contain letters,
+    % digits, and underscores, and be under a maximum length
+    if ~isvarname(col_label)
+        % Find any non-alphanumeric characters and replace with '_'
+        col_label(~(isstrprop(col_label,'alphanum'))) = '_';
+        % Check if first character is a letter, and append 'Z' if not
+        if ~(isletter(col_label(1)))
```

Comment:
I don't see a 'Z' at all, am I missing something or did the comment get out of date with the code?